### ISSUE TITLE
NullReferenceException in MD.Core.Instrumentation.TimeCounter`1[T].Trace

### ISSUE THREAD

**Issue by KirillOsenkov** [mono/monodevelop#5658]
From logs:
7.7 Preview (7.7 build 768)

```
System.NullReferenceException: Object reference not set to an instance of an object
  at MonoDevelop.Core.Instrumentation.TimeCounter`1[T].Trace (System.String message) [0x00078] in /Users/vsts/agent/2.136.1/work/1/s/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Core.Instrumentation/TimeCounter.cs:144 
  at MonoDevelop.Core.Instrumentation.TimerCounter.Trace (System.String message) [0x00010] in /Users/vsts/agent/2.136.1/work/1/s/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Core.Instrumentation/TimerCounter.cs:98 
  at MonoDevelop.Ide.Editor.Extension.CompletionTextEditorExtension+<>c.<DoHandleCodeCompletionAsync>b__34_0 (System.Threading.Tasks.Task`1[TResult] t) [0x00000] in /Users/vsts/agent/2.136.1/work/1/s/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.Extension/CompletionTextEditorExtension.cs:590 
  at System.Threading.Tasks.ContinuationTaskFromResultTask`1[TAntecedentResult].InnerInvoke () [0x00024] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-02/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/threading/Tasks/TaskContinuation.cs:159 
  at System.Threading.Tasks.Task.Execute () [0x00010] in /Users/builder/jenkins/workspace/build-package-osx-mono/2018-02/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/threading/Tasks/Task.cs:2509 
```

> VS bug [#662495](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/662495)

**Comment by KirillOsenkov** [comment]
https://github.com/mono/monodevelop/blob/9c353fd16416d806d498cf1b4e3f492c743360f1/main/src/core/MonoDevelop.Core/MonoDevelop.Core.Instrumentation/TimeCounter.cs#L144

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-19] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #5721 merged by: iain  
 Author: mono  
 Commit: 7bda5435f21086fd41de56bb805ab8c04e4eeaaa (xamarin/monodevelop)
